tests/e2e/test_rosacli_cluster_post.go

```diff
@@ -461,8 +461,55 @@ var _ = Describe("Delete BYO OIDC cluster testing",
 				Expect(err.Error()).To(ContainSubstring("NoSuchEntity"))
 
 			})
+		It("to verifiy the sts cluster is deleted successfully - [id:75256]",
+			labels.High, labels.Runtime.DestroyPost,
+			func() {
+				By("Check the operator-roles is deleted")
+				clusterDetail, err := profilehandler.ParserClusterDetail()
+				Expect(err).To(BeNil())
+
+				operatorRolesArn := clusterDetail.OperatorRoleArn
+				for _, v := range operatorRolesArn {
+					_, roleName, err := common.ParseRoleARN(v)
+					Expect(err).To(BeNil())
+					opRole, err := awsClient.IamClient.GetRole(
+						context.TODO(),
+						&iam.GetRoleInput{
+							RoleName: &roleName,
+						})
+					Expect(opRole).To(BeNil())
+					Expect(err.Error()).To(ContainSubstring("NoSuchEntity"))
+				}
+
+				By("Check the cluster is deleted")
+				clusterID = config.GetClusterID()
+				rosaClient.Runner.UnsetArgs()
+				clusterListout, err := clusterService.List()
+				Expect(err).To(BeNil())
+				clusterList, err := clusterService.ReflectClusterList(clusterListout)
+				Expect(err).To(BeNil())
+				Expect(clusterList.IsExist(clusterID)).To(BeFalse())
+
+				By("Check account roles are deleted")
+				var accountRoleArns []string
+				accountRoleArns = append(accountRoleArns, clusterConfig.Aws.Sts.RoleArn)
+				accountRoleArns = append(accountRoleArns, clusterConfig.Aws.Sts.SupportRoleArn)
+				accountRoleArns = append(accountRoleArns, clusterConfig.Aws.Sts.WorkerRoleArn)
```

Comment:
Doesn't it work like that ?
```suggestion
				accountRoleArns := []string{
				    clusterConfig.Aws.Sts.RoleArn,
				    clusterConfig.Aws.Sts.SupportRoleArn,
				    clusterConfig.Aws.Sts.WorkerRoleArn,
				}
```